packages/grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -286,6 +290,23 @@ class GridProEditColumn extends GridColumn {
     this._focusEditor(editor);
   }
 
+  _onMouseDown(editor) {
+    return (e) => {
+      this.__mouseDownDebouncer = Debouncer.debounce(this.__mouseDownDebouncer, animationFrame, () => {
+        if (e.target !== editor.inputElement) {
+          this._focusEditor(editor);
```

Comment:
Calling `_focusEditor` here to prevent one issue I noticed when I used a `NumberField` as the editor. Without this change, the user can click the +/- buttons but as the `NumberField` does not restore focus to the input element, the user could click in another cell, which would leave the editor open. 

With this change, there's going to be a different behavior in the `NumberField` element inside a `GridPro`, meaning that the focus in the input element is restored after clicking the +/- buttons.